specification/cognitiveservices/data-plane/Personalizer/preview/v1.1-preview.2/Personalizer.json

```diff
@@ -0,0 +1,1895 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.1-preview.2",
+    "title": "Personalizer Base",
+    "description": "Personalizer Service is an Azure Cognitive Service that makes it easy to target content and experiences without complex pre-analysis or cleanup of past data. Given a context and featurized content, the Personalizer Service returns which content item to show to users in rewardActionId. As rewards are sent in response to the use of rewardActionId, the reinforcement learning algorithm will improve the model and improve performance of future rank calls."
+  },
+  "basePath": "/personalizer/v1.1-preview.2",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/configurations/service": {
```

Comment:
@slahabar -- Many of the operations specified in here do not have a default response. It is good practice to ensure that defaults are indicated.

Reply:
For the new apis I have added the default response. Is this the same response that you are mentioning here? If we need to add this for all the existing apis, I will create a task on Personalizer to fix this outside this PR. 
"responses": {
          "204": {
            "description": "Success"
          },
          "default": {
            "description": "Invalid request.",
            "schema": {
              "$ref": "#/definitions/ErrorResponse"
            }
          }
        },

Reply:
As discussed in the meeting Personalizer will create a separate task to address this issue before the SDK release.